app.py

```diff
@@ -50,7 +51,21 @@ def _get_local_time():
             return f"Could not process time information. {e}", 404
 
 
-#TODO Add a route that displays the current time for all US timezones.
+@app.route('/time-difference', methods=['GET', 'POST','PUT'])
+def time_diff():
```

Comment:
It might be helpful to add your note about how to test the route in a comment here for others reading the code directly.